internal/fields/sql.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/go-sql-driver/mysql"
+	"github.com/gogo/protobuf/proto"
```

Comment:
I think we already might have a gogo proto dependency, but if not, it'd be nice to not have this edge. Can we inline the important parts of the interface?

Reply:
We do already have the dependency (it's checked into the vendor folder).  Unfortunately I don't think there's a good way to avoid adding it since we're depending on the `proto.Marshal`/`proto.Unmarshal` functions.